apps/modernization-ui/src/apps/search/useSearchResultDisplay.tsx

```diff
@@ -5,28 +5,55 @@ import { Term } from './terms';
 
 type View = 'list' | 'table';
 
+export enum SortDirection {
+    Asc = 'ASC',
+    Desc = 'DESC'
+}
+
+export enum SortField {
+    BirthTime = 'birthTime',
+    LastNm = 'lastNm',
+    Relevance = 'relevance'
+}
+
+type Sort = { sortDirection?: SortDirection; sortField: SortField };
+
```

Comment:
issue:  This needs to able to handle the sorting for both list and table views, checkout the [useSorting](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/sorting/useSorting.tsx) hook.  These components are already under a `SortProvider`

Reply:
Okay I will look into this